target/linux/ath79/dts/ar9344_mikrotik_sxt5n.dtsi

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	compatible = "mikrotik,sxt5n", "qca,ar9344";
+	model = "MikroTik SXT5N platform";
```

Comment:
This does not make much sense in a DTSI, unless for descriptive purposes. I personally tend to remove it, as it's overwritten in any case anyway.

Reply:
Yes, it's purely informational. Ubiquiti's boards/platforms have it:

```
	compatible = "ubnt,wa", "qca,ar9342";
	model = "Ubiquiti Networks WA board";
```
```
	compatible = "qca,ar7242";
	model = "Ubiquiti Networks SW board";
```
[...]
which I find convenient to indicate that this particular .dtsi file is not for a single device, but shared by a family of similar devices.

And there are also device-specific .dtsi files:
```
	compatible = "glinet,gl-ar750s", "qca,qca9563";
	model = "GL.iNet GL-AR750S";
```
```
	compatible = "tplink,tl-wr741n-v4", "qca,ar9331";
	model = "TP-Link TL-WR741N/ND v4";
```

I would like to keep it, but I don't have strong arguments for it.

Reply:
So... keeping it for informative purposes.